### PR TITLE
fix: Correct bounding box values on region move

### PR DESCRIPTION
**Pull request by wbreza**
Corrects an issue where bounding box position was not being updated correctly when a region was moved within the canvas editor.  Also randomized region test data to ensure we are tests are more accurate

### CODE REVIEW

File: src/common/mockFactory.ts [hunk 1]
```diff
@@ -577,32 +545,73 @@ export default class MockFactory {
...
+        };
+
+        return {
+            id,
             boundingBox: {
-                height: 100,
-                width: 100,
-                left: 0,
-                top: 0,
+                left: origin.x,
```
Comment (tbarlow12) [inline src/common/mockFactory.ts]: Nice

File: src/common/mockFactory.ts [hunk 2]
```diff
@@ -901,4 +910,8 @@ export default class MockFactory {
                 return StorageType.Other;
         }
     }
+
+    private static randomNumber(min: number, max: number) {
```
Comment (tbarlow12) [inline src/common/mockFactory.ts]: Not that this matters much since it's so simple, but this already exists in the `utils` file

Comment (wbreza) [reply src/common/mockFactory.ts]: Thanks - will convert to use that method.

File: src/react/components/pages/editorPage/canvas.tsx [hunk 3]
```diff
@@ -168,6 +168,12 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
 
         if (movedRegion) {
             movedRegion.points = scaledRegionData.points;
+            movedRegion.boundingBox = {
+                height: scaledRegionData.height,
```
Comment (wbreza) [inline src/react/components/pages/editorPage/canvas.tsx]: This was the core bug fix.

**Pull request merged**